File: sdkwork-app-sdk-typescript/src/types/ai-usage-stats-vo.ts
/** AI使用统计VO */
export interface AiUsageStatsVO {
  /** 创建时间 */
  createdAt?: string;
  /** 更新时间 */
  updatedAt?: string;
  /** 统计ID */
  statsId?: string;
  /** 模型名称 */
  modelName?: string;
  /** 提示词Token数 */
  promptTokens?: number;
  /** 完成Token数 */
  completionTokens?: number;
  /** 总Token数 */
  totalTokens?: number;
  /** 请求次数 */
  requestCount?: number;
  /** 错误次数 */
  errorCount?: number;
  /** 日期 */
  date?: string;
  /** 用户ID */
  userId?: string;
}

File: sdkwork-app-sdk-kotlin/src/main/kotlin/com/sdkwork/app/AiUsageStatsVO.kt
package com.sdkwork.app

data class AiUsageStatsVO(
    val createdAt: String? = null,
    val updatedAt: String? = null,
    val statsId: String? = null,
    val modelName: String? = null,
    val promptTokens: Int? = null,
    val completionTokens: Int? = null,
    val totalTokens: Int? = null,
    val requestCount: Int? = null,
    val errorCount: Int? = null,
    val date: String? = null,
    val userId: String? = null
)
